tutoroars/templates/oars/jobs/init/oars_init_schemas_tables_users.sh

```diff
@@ -56,7 +56,12 @@ CREATE MATERIALIZED VIEW IF NOT EXISTS {{ OARS_XAPI_DATABASE }}.{{ OARS_XAPI_TRA
     SELECT
     event_id as event_id,
     JSON_VALUE(event_str, '$.verb.id') as verb_id,
-    JSON_VALUE(event_str, '$.actor.account.name') as actor_id,
+    COALESCE(
+        NULLIF(JSON_VALUE(event_str, '$.actor.account.name'), ''),
+        NULLIF(JSON_VALUE(event_str, '$.actor.mbox'), ''),
+        NULLIF(JSON_VALUE(event_str, '$.actor.mbox_sha1sum'), ''),
+        'none'
```

Comment:
I think we just want it to be an empty string if nothing hits, so you should be able to get rid of this line and the last `NULLIF`.

Reply:
Corrected